.github/dependabot.yml

```
@@ -9,4 +9,4 @@ updates:
       interval: 'weekly'
     allow:
       # Starting slow by only checking our own packages for updates.
-      - dependency-name: '@metaplex-foundation/*'
+      - dependency-name: '@metaplex-foundation/*'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)